### DISCUSSION TITLE
where the new callbaclks located?

### DISCUSSION THREAD

**Discussion by JiahaoYao** [Lightning-AI/pytorch-lightning#13586] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
https://github.com/Lightning-AI/lightning/blob/master/src/pytorch_lightning/trainer/callback_hook.py

these callbacks are removed but where are the new locations for them?

**Comment by akihironitta** [comment, marked as answer]
Hi @JiahaoYao These hooks were not meant to be called by users. (related to #10575)

Can I ask what you're trying to find? Maybe, what you're looking for is user-facing `Callback`'s hooks? If so, they are located here in the codebase: https://github.com/Lightning-AI/lightning/blob/b59f80224843886459d54c828325683d770da746/src/pytorch_lightning/callbacks/callback.py

**Reply by JiahaoYao** [reply]
thanks
